docs/LLS_Metallicities-Wotta/README.md

```diff
@@ -0,0 +1,52 @@
+## Create MCMC input files
```

Comment:
Also, very helpful!  Am wondering if the folder should be 
renamed to something like MCMC-Metallicitites ?